### DISCUSSION TITLE
How to know if a Symbol is JavaDefined or Scala2Defined

### DISCUSSION THREAD

**Discussion by anjunar** [scalacenter/tasty-query#345] [Q&A] [answered]
I cannot see where I can access the class "Flags", because it is private.

**Comment by sjrd** [comment, marked as answer]
Everything stored in "flags" is exposed as specific accessors on the symbols instead. In particular, look at [`TermOrTypeSymbol.sourceLanguage`](https://javadoc.io/doc/ch.epfl.scala/tasty-query_3/latest/tastyquery.html#sourceLanguage-0).
